packages/suite/src/hooks/wallet/coinmarket/form/useCoinmarketVerifyAccount.tsx

```diff
@@ -96,6 +96,7 @@ const useCoinmarketVerifyAccount = ({
     const isDebug = useSelector(selectIsDebugModeActive);
     const device = useSelector(selectDevice);
     const dispatch = useDispatch();
+    const [isMenuOpen, setIsMenuOpen] = useState<boolean | undefined>(undefined);
```

Comment:
Maybe using just `boolean` values without `undefined` here would be sufficient. WDYT?

Reply:
It is used for one specific purpose, when `account.type === 'ADD_SUITE'` is when the modal will open and select stays opened after the action is completed in the modal. 

If the `isMenuOpen` is `undefined`, there is no need to manage the state for other cases.